Source: Timo Gurr <tgurr@exherbo.org>
Reason: Licence file not available anymore, download is done by our exheres.
Upstream: Unknown/Exherbo only patch

diff -Naur ilbc-0.6/downloads/known_checksums/gips_iLBClicense.pdf.md5 ilbc-0.6.new/downloads/known_checksums/gips_iLBClicense.pdf.md5
--- ilbc-0.6/downloads/known_checksums/gips_iLBClicense.pdf.md5	2011-06-02 01:40:16.908151932 +0200
+++ ilbc-0.6.new/downloads/known_checksums/gips_iLBClicense.pdf.md5	1970-01-01 01:00:00.000000000 +0100
@@ -1 +0,0 @@
-a3023336c7f78bf8436dbeba2b53dad4  gips_iLBClicense.pdf
diff -Naur ilbc-0.6/downloads/Makefile.am ilbc-0.6.new/downloads/Makefile.am
--- ilbc-0.6/downloads/Makefile.am	2011-06-02 01:40:16.908151932 +0200
+++ ilbc-0.6.new/downloads/Makefile.am	2011-06-02 02:30:14.356153262 +0200
@@ -6,12 +6,7 @@
 extract_script = extract-cfile.awk
 extract_script_url = http://www.ilbcfreeware.org/documentation/$(extract_script)
 
-# and for the license
-license = gips_iLBClicense.pdf
-license_url = http://www.ilbcfreeware.org/documentation/$(license)
-# license_url = http://gipscorp.com/files/english/$(license)
-
-downloaded_files = $(technical_spec) $(extract_script) $(license)
+downloaded_files = $(technical_spec) $(extract_script)
 checkmarks = $(downloaded_files:%=%.check)
 success_marker = ../src/iLBC_define.h
 BUILT_SOURCES = $(downloaded_files) $(checkmarks) $(success_marker)
@@ -20,15 +15,6 @@
 checksum_files = $(addprefix $(checksum_dir)/, $(downloaded_files:%=%.md5))
 EXTRA_DIST = $(checksum_files)
 
-$(technical_spec):
-	$(WGET) $(technical_spec_url)
-
-$(extract_script):
-	$(WGET) $(extract_script_url)
-
-$(license):
-	$(WGET) $(license_url)
-
 # for check, go to $(srcdir) or ./, depending on where file is
 $(checkmarks): %.check: %
 	cd $(<D); $(MD5SUM) -c $(checksum_dir)/$(<F).md5
